addons/account_edi_ubl_cii/models/account_move_send.py

```diff
@@ -188,6 +188,12 @@ def _hook_invoice_document_after_pdf_report_render(self, invoice, invoice_data):
         reader_buffer.close()
         writer_buffer.close()
 
+    def _get_additional_document_reference_custom_nodes(self, invoice):
```

Comment:
Wtf... this is a way too generic method to be in account_edi_ubl_cii without any prefix than in account.
When adding such method, you need to add at least a docstring...

Add a new method in account_edi_common building the `AdditionalDocumentReference` with an ir.attachment as parameter. In zatca, override it.

Reply:
1) You are right and you made me think about the fact that it's weird that `account.move.send` needs to know anything about the edi implementation details.
I've just made a commit refactoring a part of that code: `[REF] account_{edi_ubl_cii/peppol},l10n_dk_oioubl: seperate responsabilities`.
Let me know if that could be ok or if I should drop or improve it.